r/R/dataset-write.R

```diff
@@ -60,8 +62,13 @@ write_dataset <- function(dataset,
                           format = c("parquet", "feather", "arrow", "ipc"),
                           partitioning = dplyr::group_vars(dataset),
                           basename_template = paste0("part-{i}.", as.character(format)),
-                          hive_style = TRUE,
+                          hive_style = TRUE, max_partitions = 1024L,
```

Comment:
Minor: in the .R code, we should follow the style here with each argument on a new line.